### PR TITLE
change twitter icon to X

### PR DESCRIPTION
**Pull request by serena97**
Screenshot 2023-09-06 at 12 56 53

### CODE REVIEW

File: components/x-live-blog-post/src/svgIcons.jsx [hunk 1]
```diff
@@ -6,8 +6,8 @@ Once this is resolved, these components should be replaced with o-share https://
  */
 export function TwitterSVG() {
```
Comment (AliceArmstrong) [inline components/x-live-blog-post/src/svgIcons.jsx:7]: **Non-blocking suggestion:** Should we update this function name as well? Side-note, `X` is such an awkward name to use in functions 🙈

**Review by AliceArmstrong — COMMENTED** [review]
Looks good, one non-blocking suggestion!

File: components/x-live-blog-post/src/svgIcons.jsx (hunk 1 above)
Comment (AliceArmstrong) [reply components/x-live-blog-post/src/svgIcons.jsx:7]: Ah, just realise this is probably used externally, so probably a lot of effort to rename it!

**Review by AliceArmstrong — APPROVED** [review]

**Review by emortong — APPROVED** [review]
Looks good, thanks!

**Review by ivomurrell — APPROVED** [review]
**question (non-blocking)**: should we change this copy to "Share on X" now? https://github.com/Financial-Times/x-dash/blob/da2f6ade0a9c162ebe51155e5b38dad826944410/components/x-gift-article/src/SocialShareButtons.jsx#L50 https://github.com/Financial-Times/x-dash/blob/da2f6ade0a9c162ebe51155e5b38dad826944410/components/x-live-blog-post/src/ShareButtons.jsx#L37

**Pull request merged**